### PR TITLE
Add subscription delete method to examples

### PR DESCRIPTION
**Pull request by swilczynski**

### CODE REVIEW

File: openapi/code_samples/PHP/subscriptions@{id}/delete.php [hunk 1]
```diff
@@ -0,0 +1 @@
+$subscription = $client->subscriptions()->delete('subscriptionId');
```
Comment (ashkarpetin) [inline openapi/code_samples/PHP/subscriptions@{id}/delete.php:1]: I don't see `delete` method here https://github.com/Rebilly/rebilly-php/blob/374c9a761556074b98720eb46fa3800244b9fd4c/src/Services/SubscriptionService.php#L32

Comment (swilczynski) [reply openapi/code_samples/PHP/subscriptions@{id}/delete.php:1]: Should I rebuild both PHP and JS Sdks? I thought it was done automatically

Comment (ashkarpetin) [reply openapi/code_samples/PHP/subscriptions@{id}/delete.php:1]: PHP is not generated automatically, you have to add this method manually

Comment (swilczynski) [reply openapi/code_samples/PHP/subscriptions@{id}/delete.php:1]: It's here now
https://github.com/Rebilly/rebilly-php/blob/main/src/Services/SubscriptionService.php

**Review by ashkarpetin — DISMISSED** [review]

**Review by vykhrystiuk — DISMISSED** [review]

File: openapi/code_samples/JavaScript/subscriptions@{id}/delete.js [hunk 2]
```diff
@@ -0,0 +1 @@
+const invoiceItem = await api.subscriptions.delete({id: 'id'});
```
Comment (VladimirBerdnik) [inline openapi/code_samples/JavaScript/subscriptions@{id}/delete.js:1]:
```suggestion
const subscription = await api.subscriptions.delete({id: 'id'});
```

Comment (VladimirBerdnik) [inline openapi/code_samples/JavaScript/subscriptions@{id}/delete.js:1]:
```suggestion
const subscription = await api.subscriptions.delete({id: 'id'});
```

**Review by VladimirBerdnik — APPROVED** [review]

**Review by akozhevnikov-rebilly — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Pull request closed without merging**